galley/pkg/config/schema/schema.go

```diff
@@ -187,6 +187,17 @@ func (k KubeResources) MustFind(group, kind string) KubeResource {
 	return r
 }
 
+// GetDisabled returns the names of disabled collections
+func (k KubeResources) GetDisabled() collection.Names {
```

Comment:
Nit: Rename to GetDisabledCollections(), to be consisttent with other members?

Reply:
Switched to DisabledCollections (paralleling the Collections function).